packages/caver-rpc/src/klay.d.ts

```diff
@@ -0,0 +1,117 @@
+/*
+    Copyright 2021 The caver-js Authors
+    This file is part of the caver-js library.
+
+    The caver-js library is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Lesser General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    The caver-js library is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+    GNU Lesser General Public License for more details.
+
+    You should have received a copy of the GNU Lesser General Public License
+    along with the caver-js. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+import { AccountKeyFormat, AccountKeyObject } from '../../caver-core-helpers/src/formatters'
+import RpcCallToMethod, { Block, BlockWithConsensusInfo, SignResult, TransactionFromNode } from '../../caver-rtm/src'
+import AbstractFeeDelegatedTransaction from '../../caver-transaction/src/transactionTypes/abstractFeeDelegatedTransaction'
+import { PromiEvent } from '../../caver-utils/src'
+import Validator from '../../caver-validator/src'
+
+export interface DecodeAanchoringTransaction {
+    BlockHash: string
+    BlockNumber: number
+    ParentHash: string
+    TxHash: string
+    StateRootHash: string
+    ReceiptHash: string
+    BlockCount: number
+    TxCount: number
+}
+
+export interface KlaytnCall {
+    getAccountKey: KlayRPC['getAccountKey']
+    getChainId: KlayRPC['getChainId']
+    getGasPrice: KlayRPC['getGasPrice']
+    getTransactionByHash: KlayRPC['getTransactionByHash']
+    getTransactionCount: KlayRPC['getTransactionCount']
+}
+
+export class KlayValidator extends Validator {
+    static _klaytnCall: KlaytnCall
+}
+
+export default class KlayRPC {
+    constructor(...args: any[])
+
+    accountCreated: RpcCallToMethod['klay_accountCreated']
+    getAccounts: RpcCallToMethod['klay_accounts']
+    encodeAccountKey(accountKey: AccountKeyFormat): Promise<string>
```

Comment:
What is the difference between `RpcCallToMethod['xxx']` and `Promise<xxx>`?
Some rpc call's return type is defined with RpcCallToMethod and others are defined with Promise.
Is there a standard for distinguishing return types like this?

Reply:
`RpcCallToMethod` is based on `rpc.json`.

`klay.js` is creating methods like below
```
new Method({
    name: 'accountCreated',
    call: 'klay_accountCreated',
    params: 2,
    inputFormatter: [formatters.inputAddressFormatter, formatters.inputDefaultBlockNumberFormatter],
}),
new Method({
    name: 'getAccounts',
    call: 'klay_accounts',
    params: 0,
}),
...
```

But, `encodeAccountKey(klay_encodeAccountKey)`, `decodeAccountKey(klay_decodeAccountKey)`, `getBlockByHash(klay_getBlockByHash)`... are not defined in `rpc.json`
Therefore, I wrote an additional definition.

Reply:
Oh, okay Thank you for detail explaination :) 
However, since the `caver-rpc` package no longer uses `rpc.json`, it would be better if other rpc calls were defined like the way `encodeAccountKey` is defined in the klay.d.ts file.

Reply:
What is different with `caver-rpc` and `caver-rtm`? I think they do the same thing that call RPC to same endpoint.
(Some were added separately to `caver-rpc`. `caver-rpc`⊃`caver-rtm` I guess.)
If separate them, code will be duplicated, it will be difficult to maintain.